apps/dsp-app/src/app/app-routing.module.ts

```diff
@@ -42,10 +43,19 @@ const routes: Routes = [
         path: 'login',
         component: LoginFormComponent,
     },
+    {
+        path: 'project/create-new',
```

Comment:
I think project should be plural `projects` here, right?

Reply:
In the end, yes. But I set it to "project" intentionally. The reasons are:
- the route "projects" is currently occupied by the UserComponent in 163, which is a strange parent route
- the projects which we have in mind is currently routed at "system/projects" in 183

So for this PR I decided to not change all routes to what is already defined, because what is defined for now is not yet final, not yet applicable. Changing to the desired "projects" route would possibly imply to snowball through all routes and change all routing to a stage which is not yet final. It would possibly imply to change and test a lot of navigation. This should imho be done in a separate PR/issue. What do you think?

Reply:
Edit: The goal was to make this component routable/standalone in order to refactor it.